### ISSUE TITLE
test suite: EDQUOT is not expanded in quota.kshlib

### ISSUE THREAD

**Issue by avg-I** [zfsonlinux/zfs#4443]
I am getting the following error on Gentoo (ksh-93.20140625):
```
/usr/share/zfs/zfs-tests/tests/functional/quota/quota_001_pos.ksh[74]: log_must[71]: log_pos[182]: fill_quota: line 56: EDQUOT: arithmetic syntax error
````

**Comment by behlendorf** [comment]
Shouldn't this be `$EDQUOT`?

**Comment by avg-I** [comment]
Yes, this should be trivial to fix.  In fact, I already have a patch, will create a PR.

**Comment by behlendorf** [comment]
Awesome, it saw you opened a few other issues regarding the ZFS Test Suite I'd love to see fixes for. :)

**Comment by behlendorf** [comment]
Closing, addressed by commit bebd73f2c3822a86863e7508e7a3e2929d871113
